frontend/src/components/HospitalInventoryList.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import ListGroup from 'react-bootstrap/ListGroup'
+import Badge from 'react-bootstrap/Badge'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+export default () => {
+  
+  return (
+    <div>
+      <h1>Fortis Hospital</h1>
+      <h5>Inventory List</h5>
+      <ListGroup>
+        <ListGroup.Item>
```

Comment:
Let's make it a bit closer to the requirements by involving a fake API call. We don't yet have any API for that, but I already sent out specs (see https://gist.github.com/bl-nero/6354aeabf46f601ac6a9581b00ba377d), so you can see what we can expect. Then go to `apis/infomed.tsx` file, see how I created some fake API calls, and use it to create a function that will make a fake call to the hospital inventory API. For the models, it should be enough to declare them as simple TypeScript interfaces, possibly looking exactly like the ones returned by the the API as specified.

After that, use the `useEffect` hook (see a similar approach in `components/AppointmentScheduler.tsx`) to fetch the items.

Then you can replace the long list of list items here with a loop over the data that's been fetched from the API.